pkg/plugins/golang/v2/api.go

```diff
@@ -198,8 +198,8 @@ func (p *createAPISubcommand) PostScaffold() error {
 		return fmt.Errorf("unknown pattern %q", p.pattern)
 	}
 
-	if p.runMake {
-		return util.RunCmd("Running make", "make")
+	if p.runMake { // TODO: check if API was scaffolded
+		return util.RunCmd("Running make", "make", "generate")
```

Comment:
Should we not run make manifests as well by default?

Reply:
Make manifest requires the user to first edit the `*_types.go` file and rerun it, and you get no advantage of running it earlier. `make generate` advantage is minimal but I kept it.

Reply:
cacher.